### PR TITLE
cos template update for charts 2.2.24

### PR DESCRIPTION
**Pull request by Bhagyashreek8**

### CODE REVIEW

File: config-templates/ibm/ibm-object-storage-plugin/2.2/deployment.yaml [hunk 1]
```diff
@@ -4,7 +4,7 @@ metadata:
   name: ibm-object-storage-plugin
   namespace: kube-system
   annotations:
-    revision: 18  ## On every new release of the ibm-object-storage-plugin helm-chart revision should be updated to show users new version for cos-s3fs is available.
+    revision: 189 ## On every new release of the ibm-object-storage-plugin helm-chart revision should be updated to show users new version for cos-s3fs is available.
```
Comment (mssachan) [inline config-templates/ibm/ibm-object-storage-plugin/2.2/deployment.yaml]: Fix the revision `189` --> `19`

File: config-templates/ibm/ibm-object-storage-plugin/changelog.json [hunk 2]
```diff
@@ -3,25 +3,24 @@
         "2.2": {
             "status": "supported",
             "latest-revision": {
-                "revision": "18",
-                "release-date": "02 February 2024",
-                "cves": "CVE-2023-3446, CVE-2023-3817, CVE-2023-5678",
+                "revision": "19",
+                "release-date": "29 February 2024",
+                "cves": "",
```
Comment (mssachan) [inline config-templates/ibm/ibm-object-storage-plugin/changelog.json:8]: Please retain `"ubi": "",` field. Somehow we missed in last release.

File: config-templates/ibm/ibm-object-storage-plugin/changelog.json [hunk 3]
```diff
@@ -3,25 +3,24 @@
...
                 "golang": "1.20.13",
```
Comment (mssachan) [inline config-templates/ibm/ibm-object-storage-plugin/changelog.json]: Also check the travis build which built the new images and get the golang used for building image from Travis.
Update golang here accordingly.

**Review by mssachan — CHANGES_REQUESTED** [review]

File: config-templates/ibm/ibm-object-storage-plugin/changelog.json (hunk 2 above)
Comment (Bhagyashreek8) [reply config-templates/ibm/ibm-object-storage-plugin/changelog.json:8]: As of now base images are not based on UBI , so ubi version will be irrelevant here.

**Review by mssachan — APPROVED** [review]
LGTM

**Pull request merged**